pkgs/applications/version-management/srcml/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, cmake, libxml2, libxslt, boost, libarchive, python, antlr }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "srcml";
```

Comment:
Can you explain what benefit this layer of indirection provides. I do not see, where `pname` is used.